### PR TITLE
Datalab: enable HTTPS for apt repos.

### PR DESCRIPTION
**Pull request by functicons**

### CODE REVIEW

File: datalab/datalab.sh [hunk 1]
```diff
@@ -115,14 +115,16 @@ function configure_master() {
   cat <<EOF >Dockerfile
 FROM ${DOCKER_IMAGE}
 
+# Enabling APT to download from HTTPS repository.
+RUN apt-get update
+RUN apt-get install -y dirmngr apt-transport-https software-properties-common
```
Comment (medb) [inline datalab/datalab.sh:120]: Why do we need `dirmngr`?

Comment (functicons) [reply datalab/datalab.sh]: Removed

**Pull request merged**